### PR TITLE
Clean up front page copy

### PR DESCRIPTION
**Pull request by ericholscher**
---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--11705.org.readthedocs.build/en/11705/

- Developer's documentation (`dev`): https://dev--11705.org.readthedocs.build/en/11705/

### CODE REVIEW

File: docs/user/index.rst [hunk 1]
```diff
@@ -221,16 +222,16 @@ Keep your documentation up to date:
 Business features
 -----------------
 
-Features for organizations and businesses:
+Our Business hosting has everything your business needs:
 
 :doc:`/commercial/organizations`
-  Learn how to manage your organization on Read the Docs.
+  Control permissions across multiple teams.
```
Comment (humitos) [inline docs/user/index.rst:228]: I'm not sure if it's better or not, but it's more natural to me the usage of _manage_ 🤷🏼 

```suggestion
  Manage permissions across multiple teams.
```

**Review by humitos — APPROVED** [review]
I like this clean up -- it has a little more marketing voice 😄

File: docs/user/index.rst [hunk 2]
```diff
@@ -124,10 +124,11 @@ Read the Docs: documentation simplified
...
    <a style="display: none;" rel="me" href="https://fosstodon.org/@readthedocs">Mastodon</a>
 
-Read the Docs simplifies software documentation
-by building, versioning, and hosting of your docs, automatically.
+Read the Docs simplifies managing software documentation
+by building and hosting your docs automatically,
+using the Git workflow you already use for code.
 Treating documentation like code keeps your team in the same tools,
-and your documentation up to date.
+and makes keeping your docs updated easier.
```
Comment (plaindocs) [inline docs/user/index.rst:131]: How about 

> Treating documentation like code lets your team use tools they already know,
and makes keeping your docs updated easier.

File: docs/user/index.rst [hunk 3]
```diff
@@ -157,7 +158,7 @@ We have a few places for you to get started:
 .. descriptions here are active
 
 :doc:`/tutorial/index`
-  Take the first practical steps with Read the Docs.
+  Take the first steps with Read the Docs in 30 minutes.
```
Comment (plaindocs) [inline docs/user/index.rst:161]: This implies to me that you can't do anything in less than 30 minutes, which isn't ideal.

```suggestion
  Follow the Read the Docs tutorial.
```

File: docs/user/index.rst [hunk 4]
```diff
@@ -171,15 +172,15 @@ Start with the basics of setting up your project:
   Learn how to configure your project with a ``.readthedocs.yaml`` file.
 
 :doc:`/guides/reproducible-builds`
-  Learn how to make your builds reproducible.
+  Keep your builds working over time by making them reproducible.
```
Comment (plaindocs) [inline docs/user/index.rst:175]:
```suggestion
  Keep your builds working by making them reproducible.
```

or keep your builds green :-p

File: docs/user/index.rst [hunk 5]
```diff
@@ -190,24 +191,24 @@ Hosting documentation
 Learn more about our hosting features:
 
 :doc:`/versions`
-  Host multiple versions of your documentation.
+  Give your readers multiple versions of your documentation.
```
Comment (plaindocs) [inline docs/user/index.rst:194]:
```suggestion
  Show your readers multiple versions of your documentation.
```

maybe?

File: docs/user/index.rst [hunk 6]
```diff
@@ -190,24 +191,24 @@ Hosting documentation
...
 :doc:`/localization`
-  Host your documentation in multiple languages.
+  Translate your documentation into multiple languages.
 
 :doc:`/versioning-schemes`
   Learn about different URL versioning schemes.
 
 :doc:`/custom-domains`
-  Host your documentation on your own domain.
+  Brand your documentation on your own domain.
```
Comment (plaindocs) [inline docs/user/index.rst:207]: Love this!

File: docs/user/index.rst [hunk 7]
```diff
@@ -221,16 +222,16 @@ Keep your documentation up to date:
...
-Features for organizations and businesses:
+Our Business hosting has everything your business needs:
 
 :doc:`/commercial/organizations`
-  Learn how to manage your organization on Read the Docs.
+  Control permissions across multiple teams.
 
 :doc:`/commercial/single-sign-on`
-  Learn how to use single sign-on with Read the Docs.
+  Stay secure with Single Sign On.
```
Comment (plaindocs) [inline docs/user/index.rst:231]:
```suggestion
  Improve organization security with Single Sign On.
```

maybe? or

> Manage Read the Docs access with Single Sign On? 

(though that kinda implies docs access not edit access)

File: docs/user/index.rst [hunk 8]
```diff
@@ -221,16 +222,16 @@ Keep your documentation up to date:
...
 
 :doc:`/commercial/sharing`
-  Learn how to share your documentation with others.
+  Share your docs easily with contractors or customers.
```
Comment (plaindocs) [inline docs/user/index.rst:234]:
```suggestion
  Share your private docs easily with contractors or customers.
```

**Review by plaindocs — APPROVED** [review]
This is great :+1: :+1:

**Comment by ericholscher** [comment]
I took applied most of the comments, and merging :+1:

**Pull request merged**